uniffi_core/src/ffi/rustfuture.rs

```diff
@@ -434,7 +434,7 @@ mod tests {
         fn rust_future_weak(&self) -> Weak<TestRustFuture> {
             // It seems like there's not a great way to get an &Arc from a Pin<Arc>, so we need to
```

Comment:
is this comment still correct?

Reply:
heh - well, I guess the comment probably remains correct, but I mean "relevant" :)

Reply:
Comment removed :-).